### ISSUE TITLE
Publish sass.js compiled with exceptions

### ISSUE THREAD

**Issue by afiestas** [medialize/sass.js#23]
Hey, could you publish a compiled version of sass.js with exceptions enabled?

Thanks!

**Comment by rodneyrehm** [comment]
I'm not sure what you want me to do here. The [Makefile patch](https://github.com/medialize/sass.js/blob/master/libsass/Makefile.patch#L16) already includes [`-s DISABLE_EXCEPTION_CATCHING=0`](http://kripken.github.io/emscripten-site/docs/optimizing/Optimizing-Code.html#optimizing-code-exception-catching).

**Comment by afiestas** [comment]
What I meant is, the bower artifact contains a libsass compiled without exceptions: https://github.com/medialize/sass.js/blob/master/dist/sass.js

With that copy (that is also deployed in http://medialize.github.io/sass.js/) when a compilation fails all we get is:
Unknown Error: you need to compile libsass.js with exceptions to get proper error messages

Thanks.

**Comment by rodneyrehm** [comment]
I see what you mean. You're saying that you're not getting the actual error messages as returned by libsass exposed in JS.

When compiling invalid scss:

```js
Sass.compile(".m { width: $foobar; }");
```

internally the following error is raised and caught:

```
"TypeError: Cannot read property '0' of null
    at Object.Sass.compile (http://test.dev/medialize/sass.js/src/sass.js:149:47)
    at <anonymous>:2:6
    at Object.InjectedScript._evaluateOn (<anonymous>:777:140)
    at Object.InjectedScript._evaluateAndWrap (<anonymous>:710:34)
    at Object.InjectedScript.evaluate (<anonymous>:626:21)"
```

externally you get the following error object:

```
{
  line: null,
  message: "Unknown Error: you need to compile libsass.js with exceptions to get proper error messages"
}
```

The culprit is within sass.js itself. When originally written, errors were returned in a different format than they are now. [Line 148](https://github.com/medialize/sass.js/blob/master/src/sass.js#L148) was responsible for cleaning the error message. An it is now the reason things are failing.

What was added originially because I did not fully understand what emscripten was doing with the C++ Exceptions, was now conceiling an error in my own code because libsass changed the syntax of its errors.

The is fixed and released as [v0.6.3](https://github.com/medialize/sass.js/releases/tag/v0.6.3)
